css/main.less

```diff
@@ -58,12 +58,16 @@ p, a, li {
   color: @wet-asphalt;
 }
 
+.landingPage-title {
```

Comment:
This violates [the CSS class naming convention](https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Css.adoc#classes).